Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -109,7 +114,7 @@
     <constant name="ExtBarHCal2_rmax" value="4590*mm"/>
     <constant name="ExtBarHCal1_dz" value="150*mm"/>
     <constant name="ExtBarHCal1_zOffset" value="5000*mm + ExtBarHCal1_dz"/>
-    <constant name="ExtBarHCal2_dz" value="1350*mm"/>
+    <constant name="ExtBarHCal2_dz" value="1600*mm"/> <!-- changed to increase HCAL EB by 50cm from 1350 to 1600 -->
```

Comment:
no, you are right but the offset is calculated taking the increased dz into account (in the next line, ExtBarHCal2_zOffset). The ExtBarHCal1_zOffset corresponds to the 1st EB part, which stays as it is.